spec/models/manageiq/providers/vmware/cloud_manager/orchestration_template_spec.rb

```diff
@@ -65,6 +65,7 @@
           :memory_mb        => 2048,
           :admin_password   => nil,
           :admin_reset      => false,
+          :guest_customization => false,
```

Comment:
I'm wondering why rubocop didn't complain about `=>` alignment. I'd align it anyway.